test/tests/functional/pbs_trillion_jobid.py

```diff
@@ -165,28 +165,31 @@ def submit_job(self, sleep=10, lower=0,
         :param job_msg : Expected message upon submission failure
         :type  job_msg : int
 
+        :param running : Checks Job status R
+        :type  running : boolean(True/False)
+
         """
         arr_flag = False
         j = Job(TEST_USER)
         if((lower >= 0) and (upper > lower)):
             j.set_attributes({ATTR_J: '%d-%d' % (lower, upper)})
             arr_flag = True
-            total_jobs = upper - lower + 1
+            arr_jobs = {'total_jobs': '%d' % (upper - lower + 1)}
```

Comment:
why did you turn this into a dict? looks like you only ever store one key-val pair into it

Reply:
It's always wise to do whatever Arun tells you to :) but I think you misunderstood what he meant to say. I think he was just asking you to pass total_jobs directly to the attribute dictionary which self.server.expect() takes as its second argument, instead of passing it as a string. i.e - he asked you to do
self.server.expect(JOB, {'job_state=R': total_jobs}, ..)
instead of 
{'job_state=R': '%d' % (total_jobs)}, ..)
which you had previously.

Reply:
Addressed!